scripts/install-ilab-with-cuda.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/env bash
+
+# Before calling this script, we must set an alias for `python` like so:
```

Comment:
aliases are not inherited by exec()ed processes. It should be an exported variable.